libs/validation-constraints/src/lib/constraints/is-url.ts

```diff
@@ -1,4 +1,22 @@
-import { isString, isURL as isStringAValidUrl } from 'class-validator';
+import { isString } from './is-string';
+
+const allowedProtocols = ['http:', 'https:'];
```

Comment:
If for some reason we used ftp or some other protocol, can we just add it here?

Reply:
Yes. But you should add a failing test case, then pass the test by adding it here. The tests document the use cases.